### PR TITLE
make env vars proper for newrelic

### PR DESCRIPTION
**Pull request by timothy-spencer**
Somehow I forgot to push this on my last tspencer/newrelic branch.

### CODE REVIEW

File: config/newrelic.yml [hunk 1]
```diff
@@ -10,11 +10,11 @@
 
 common: &default_settings
   # Required license key associated with your New Relic account.
-  license_key: ENV["NEWRELIC_LICENSE"]
+  # license_key: ENV["NEW_RELIC_LICENSE_KEY"]
```
Comment (zachmargolis) [inline config/newrelic.yml:13]: do we want this commented out?

Comment (timothy-spencer) [reply config/newrelic.yml:13]: Yes, they are not actually evaluated, as it's in YAML.  I left them there to remind future me that this was where they came from.  Could change the format so it's less confusing, though.

Comment (timothy-spencer) [reply config/newrelic.yml:13]: Or delete them.  What do you like best?

Comment (zachmargolis) [reply config/newrelic.yml:13]: let's keep then but change the formatting so they don't look like commented out keys

**Review by zachmargolis — APPROVED** [review]

**Pull request merged**